pyvelox/test/test_vector.py

```diff
@@ -55,6 +55,31 @@ def test_from_list(self):
         with self.assertRaises(ValueError):
             pv.from_list([])
 
+    def test_from_list_with_type(self):
+        list_a = [0, 1, 3]
+        a = pv.from_list(list_a, pv.BooleanType())
```

Comment:
Does the function signature look good? Or will it be better if the data type is specified by a keyword argument?

Ref: Numpy array creation by data type: https://numpy.org/doc/stable/user/basics.creation.html
Torch tensor creation by data type: https://pytorch.org/docs/stable/tensors.html#initializing-and-basic-operations

Reply:
Its an interesting semantic choice you bring up - would it be possible to support both position and keyword  ?

Reply:
Made the change. It now supports passing the data type both as a positional and a keyword argument. Right now, the default value is None, and if it is not passed, the data type is inferred.